code/game/objects/items/stacks/medical.dm

```diff
@@ -127,6 +127,21 @@
 	self_delay = 20
 	max_amount = 12
 
+/obj/item/stack/medical/gauze/wirecutter_act(mob/living/user, obj/item/I)
+	if(get_amount() < 2)
+		to_chat(user, "<span class='warning'>You need at least two gauze to do this!</span>")
+		return
+	var/obj/item/stack/sheet/cloth/new_item = new(usr.loc)
+	user.visible_message("[user] cuts [src] into pieces of cloth with [I].", \
+				 "<span class='notice'>You cut [src] into pieces of cloth with [I].</span>", \
+				 "<span class='italics'>You hear cutting.</span>")
+	var/obj/item/stack/medical/gauze/R = src
+	src = null
+	var/replace = (user.get_inactive_held_item()==R)
+	R.use(2)
+	if (!QDELETED(R) && replace)
+		user.put_in_hands(new_item)

```

Comment:
Surprised this needed so many changes since I've used the code for deconstructing rods into metal as a template, but I guess it's old code. I don't really know how to fix it so I'll just get rid of it.